recipes/mob_suite/meta.yaml

```diff
@@ -1,42 +0,0 @@
-{% set version = "1.4.9.1" %}
```

Comment:
Hi, why do you remove this recipe?

Reply:
When I was working on this tool I've created a new branch and thought to remove mob_suite  hoping that master branch at merge will only add new changes. Now I realized that I need an exact copy of the existing master branch plus my new recipe. 

Yes, I still need mob_suite in there. Thank you for catching this critical error. Otherwise we might of removed the mob_suite from the repertoire of bioconda which we do not want at all.

I've added back the recipe from the mob_suite branch back to this commit.